tools/memory_estimator/template/report.json.template

```diff
@@ -0,0 +1,14 @@
+{
+    "lib":{
+        "files":[
+        ],
+        "total":{
+            "total_header":"Total estimate (configurations defined <a href=\"https://github.com/FreeRTOS/FreeRTOS/tree/lts-development/tools\" target=\"_blank\" rel=\"noopener noreferrer\">here</a>)"
```

Comment:
Hi, can we update the link to "https://github.com/FreeRTOS/FreeRTOS/tree/lts-development/tools/memory_estimator"

Reply:
Shouldn't it be https://github.com/FreeRTOS/FreeRTOS/tree/lts-development/tools/memory_estimator/config_files?